src/hooks/siteDashboardHooks/useGetSiteLaunchStatus.ts

```diff
@@ -6,6 +6,8 @@ import * as SiteLaunchService from "services/SiteLaunchService"
 
 import { SiteLaunchDto } from "types/siteLaunch"
 
+const FIVE_MINUTES = 1000 * 60 * 5
```

Comment:
suggest calling this `SITE_LAUNCH_REFETCH_INTERVAL` instead of tying it to the time. we can comment that this is 5 mins if required.

Reply:
8ee21805f49692b9254cbd47cba168b39aabaf2c